src/test/java/core/basesyntax/service/impl/ReadDataServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.service.ReadDataService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReadDataServiceImplTest {
+    private static final String FILE_DATE_NAME = "src/main/java/resources/FruitShop.csv";
```

Comment:
It's good practice to have separate resources for tests. create a new resource file in the test module. it may contain fewer transactions but assert that all lines were parsed, not just a random one